federation/executor_test.go

```diff
@@ -22,6 +22,8 @@ func createExecutorWithFederatedUser() (*Executor, *schemabuilder.Schema, *schem
 				id: int64!
```

Comment:
Do you think we need a test case to test the key validatoin?

Reply:
added!